ops/popp-tools/public/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width,initial-scale=1">
+
+  <title>3Box Test</title>
+
+	<link rel="stylesheet" href="build/bundle.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+
+  <script defer src="https://unpkg.com/3box/dist/3box.min.js"></script>
+	<script defer src="build/bundle.js"></script>
+
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet"> 
+
+</head>
+
+<body></body>
+</html>
```

Comment:
^ reindent to 2 spaces

Reply:
Done in https://github.com/gitcoinco/web/pull/9032/commits/5ad8ee2c83ecf7999f6cf29cde5afa1406edc4a3